integrations/linear/linkTemplate.vrl

```diff
@@ -1,5 +1,11 @@
 webhookId = to_string!(.webhookId)
 webhookUrl = to_string!(.webhookUrl)
-clientId = to_string!(.vars.clientId)
+env = to_string!(.env)
```

Comment:
**issue:** this will break current dashboard UI; update the dashboard ui before merging this PR

Reply:
no :) as shown